packages/components/src/speechToText.ts

```diff
@@ -3,12 +3,15 @@ import { getCredentialData } from './utils'
 import { type ClientOptions, OpenAIClient, toFile } from '@langchain/openai'
 import { AssemblyAI } from 'assemblyai'
 import { getFileFromStorage } from './storageUtils'
+import axios from 'axios'
+import FormData from 'form-data'
```

Comment:
i think this is probably redundant, I've always just directly use the built in `FormData`, can you try to remove this and see if it works fine?

Reply:
Sure, but when I remove that import, I get this error:
![Screenshot 2024-12-17 at 7 57 07 PM](https://github.com/user-attachments/assets/32309db6-6c94-4f76-b946-bf233d29362e)

Reply:
fixed